arm9/src/unk_0201CBEC.c

```diff
@@ -0,0 +1,12 @@
+#include "global.h"
+#include "fx.h"
+#include "unk_0201CBEC.h"
+
+THUMB_FUNC int FUN_0201CBEC(int x1, int y1, int x2, int y2) 
+{
+    // Calculate distance between two points with pythagoras
+    int x_delta = x1 - x2;
+    int y_delta = y1 - y2;
+
+    return FX_Sqrt((x_delta * x_delta + y_delta * y_delta) << 0xc) >> 0xc;
```

Comment:
a cast would be preferable to the shifts here

Reply:
any idea what the cast would be? I'm not sure as it's a 12 bit shift with the left shift is applied to the function parameter and the right shift to it's return?

Reply:
looking at the assembly it might not be a cast after all, as it's an asr not an lsr, and the shift is after the call, which makes it weird. it should be fine as it is then